packages/stream_thumbnail/android/src/main/kotlin/io/getstream/stream_thumbnail/StreamThumbnailPlugin.kt

```
@@ -0,0 +1,403 @@
+                            video,
+                            headers,
+                            path,
+                            format,
+                            maxh,
+                            maxw,
+                            timeMs,
+                            quality
+                        )
+                    )
+                }
+            } catch (e: IOException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)